designsystem/src/main/java/com/b1nd/dodam/designsystem/component/Card.kt

```diff
@@ -0,0 +1,260 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.b1nd.dodam.designsystem.model.AskStatus
+import com.b1nd.dodam.designsystem.theme.DodamTheme
+
+@Composable
+fun DodamAskCard(
+    askStatus: AskStatus,
+    labelText: String? = null,
+    startTime: String = "",
+    startTimeText: String = "",
+    endTime: String = "",
+    endTimeText: String = "",
+    currentLeftTime: String = "",
+    reason: String = "",
+    rejectedReason: String? = null,
+    phoneReason: String? = null,
+    progress: Float = 0.0f,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clip(shape = RoundedCornerShape(18.dp))
+            .background(MaterialTheme.colorScheme.background)
+            .padding(16.dp),
+    ) {
+        val (statusColor, statusText) = when (askStatus) {
+            AskStatus.ALLOWED -> MaterialTheme.colorScheme.primary to "승인됨"
+            AskStatus.PENDING -> MaterialTheme.colorScheme.tertiary to "대기중"
+            AskStatus.REJECTED -> MaterialTheme.colorScheme.error to "거절됨"
+        }
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Box(
+                modifier = Modifier
+                    .clip(shape = RoundedCornerShape(100))
+                    .background(statusColor)
+                    .padding(
+                        vertical = 4.dp,
+                        horizontal = 12.dp,
+                    ),
+            ) {
+                Text(
+                    text = statusText,
+                    style = MaterialTheme.typography.titleSmall,
+                    color = MaterialTheme.colorScheme.background,
+                )
+            }
+            Spacer(modifier = Modifier.weight(1f))
+            labelText?.let {
+                Text(
+                    text = labelText,
+                    style = MaterialTheme.typography.labelLarge,
+                    color = MaterialTheme.colorScheme.tertiary,
+                )
+            }
+        }
+        Spacer(modifier = Modifier.height(12.dp))
+        Text(
+            text = reason,
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onBackground,
```

Comment:
Please change the background color to MaterialTheme.colorScheme.onSurfaceVariant